src/task/task.module.ts

```diff
@@ -1,9 +1,10 @@
 import { Module } from '@nestjs/common';
+import { PinoService } from '../logger';
 import { TaskController } from './task.controller';
 import { TaskService } from './task.service';
 
 @Module({
-  providers: [TaskService],
+  providers: [TaskService, PinoService],
```

Comment:
```suggestion
  providers: [TaskService],
```
Пино сервис не нужен